### PR TITLE
Improve Titlebar, home page content on App. Improve Readme

### PR DESCRIPTION
**Pull request by debanjum**
#### Code Maintenance
  - Update dependent package, `minimist` to fix vulnerability using `npm audit fix`
  - Update, Organize Readme
    Add Deployment Badge, Project Subtitle

#### App UX Improvements
  - Add Titlebar, Improve Titlebar Styling
  - Add Content to Home Page

### CODE REVIEW

**Comment by sabaimran** [comment]
This is how the view looks in the deployment on Firefox on Ubuntu:
image

- Emoji isn't quite loading right?
- I'd suggest manipulating the div width + using line wrap in the description to put the bullet point descriptions on new lines

File: src/pages/Home.svelte [hunk 1]
```diff
@@ -4,18 +4,37 @@
     This is the home page for the application. It doesn't really do much, but it's a good place to start.
  -->
 
+<p class="description">
+    <b class="section-title">Summary</b><br />
```
Comment (sabaimran) [inline src/pages/Home.svelte:8]: Calling this section "Summary" is a bit too on-the-nose. What if we called it "Micro-dosing Randomness" or something catchy?

Alternatively, can nix the section title.

File: src/pages/Home.svelte [hunk 2]
```diff
@@ -4,18 +4,37 @@
...
-        font-size: 30px;
-        padding: 12px;
-    }
+<p class="description">
+    <b class="section-title">Details</b><br />
+    Leave some decisions to randomness to encourage exploration and reduce decision fatigue.
+
+    <ul>
+        <li>
+            If you want to explore a city, randomize the next bookstore or park you visit.<br />
```
Comment (sabaimran) [inline src/pages/Home.svelte:20]: As mentioned in other comment, suggested constraining section size using width property + using line wrap.

File: src/pages/Home.svelte [hunk 3]
```diff
@@ -4,18 +4,37 @@
...
+    Welcome to Meandering Fellow! Use it to explore your <a href=/map>neighborhood</a> <a href=/map/restaurant>restaurants</a>, <a href="/map/bar">bars</a>, <a href="/map/coffee">coffee shops</a>, <a href="/map/park">parks</a> or <a href="/map/bookstore">bookstores</a>.<br />
+</p>
 
-<style>
-    .header {
-        font-size: 30px;
-        padding: 12px;
-    }
+<p class="description">
+    <b class="section-title">Details</b><br />
```
Comment (sabaimran) [inline src/pages/Home.svelte:15]: Similar to "Summary", "Details" is a bit too spec-ish. What about "Why should I randomize?" or "Why We Built This" or "When to Use"?

File: src/pages/Home.svelte [hunk 4]
```diff
@@ -4,18 +4,37 @@
     This is the home page for the application. It doesn't really do much, but it's a good place to start.
  -->
 
+<p class="description">
+    <b class="section-title">Summary</b><br />
+    <i>Increase Exploration, Reduce Decisions 🚶🏽‍♀</i><br />
```
Comment (sabaimran) [inline src/pages/Home.svelte:9]: You should be able to directly use 🚶‍♀️ in the code.

File: src/pages/Home.svelte (hunk 1 above)
Comment (debanjum) [reply src/pages/Home.svelte]: Haha nice title suggestion. Fixed that

File: src/pages/Home.svelte (hunk 2 above)
Comment (debanjum) [reply src/pages/Home.svelte]: Used css grid to wrap lines

**Review by sabaimran — APPROVED** [review]

**Pull request merged**